src/BarChart/RenderStackBars.tsx

```diff
@@ -136,22 +136,25 @@ const RenderStackBars = (props: Props) => {
   }
   const disablePress = props.disablePress || false;
   const renderLabel = (label: String, labelTextStyle: any) => {
+    const lowestBarPosition = getLowestPosition();
     return (
       <View
         style={[
           {
             width:
               (item.stacks[0].barWidth || props.barWidth || 30) + spacing / 2,
             position: 'absolute',
-            bottom: rotateLabel ? -40 : -6 - xAxisTextNumberOfLines * 18,
+            bottom: rotateLabel
+              ? -40
+              : -6 - xAxisTextNumberOfLines * 18 + lowestBarPosition,
```

Comment:
Adding `lowestBarPosition` here to move bar label below the bar.
Otherwise the labels would be on top of the bars (in case of negative values)